astro.config.mjs

```diff
@@ -64,8 +64,7 @@ const site = 'https://v2.tauri.app';
 // https://astro.build/config
 export default defineConfig({
   site,
-  // TODO: Fix trailing slashes throughout the docs
-  // trailingSlash: 'always',
```

Comment:
should we remove this?

Reply:
this is what fixed the missing blog posts trailing slashes because it is handled by the starlight-blog, plus enabling it, makes the starlight-links-validator checks for missing trailing slashes.

Here is what I could guess after looking to the [commits history](https://github.com/tauri-apps/tauri-docs/commits/v2/astro.config.mjs): @simonhyll was doing docs work and enabled it, then "OH NO!", hundreds broken links that was too much to handle at the moment and wasn't priority....
![image](https://github.com/user-attachments/assets/2dbb65d4-b5f7-4c34-b31b-3ef7ed79e70c)

it's just a guess, but this makes sense doesn't it?